src/main/scala/is/hail/annotations/aggregators/RegionValueAggregator.scala

```diff
@@ -4,6 +4,14 @@ import is.hail.annotations._
 import is.hail.expr.types._
 
 trait RegionValueAggregator extends Serializable {
+  var inputTyp: Type = _
+  var resultTyp: Type = _
```

Comment:
is this used?

Reply:
I added resultTyp to prototype infrastructure for the groupBy aggregator.